bool FingersClose
float64 FingersTorque